### ISSUE TITLE
[ansible/postgres] No package matching 'python3-psycopg2' on RHEL 7.8

### ISSUE THREAD

**Issue by wdrury-uk** [jfrog/JFrog-Cloud-Installers#57]
Hi

Using collection - jfrog.installers:1.1.2'

Trying to use collection on a local virtualbox lab and on our RHEL 7.8 VM install runs to this point and fails trying to install a package no available.  Is this known error in the latest collection release?

```
TASK [jfrog.installers.postgres : install python3 psycopg2] ******************************************************************
fatal: [postgres]: FAILED! => changed=false
  msg: No package matching 'python3-psycopg2' found available, installed or updated
  rc: 126
  results:
  - No package matching 'python3-psycopg2' found available, installed or updated
```

**Comment by wdrury-uk** [comment]
I had to manually enable the epel-release repo on RHEL 7.8 destination database node.

   # yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm

Looking at your collection role for postgres this should have happen, but I  notice in the Ansible output log it was skipped.  Looks like it failed to identify this was running against a Red Hat Enterprise Linux distribution.

TASK [jfrog.installers.postgres : install EPEL repository] *******************************************************************
skipping: [postgres]

**Comment by chukka** [comment]
Thanks @wdrury-uk  for reporting , we currently support ubuntu only, we are planning to support RHEL, Centos 7.x /8.x in upcoming weeks (mostly next month)

Would you mind raising this change as Pull Request ?

**Comment by wdrury-uk** [comment]
Is there an update please as new version of the collection advised on my other issues was December 2020.  Whats the new ETA of the next collection version fixing this issue?

**Comment by chukka** [comment]
@wdrury-uk we are planning to release 2.x of collection within couple of weeks , all these open issues will be resloved

**Comment by wdrury-uk** [comment]
Thanks @chukka for the update.  We will plan our next internal platform release to coincide few week sprint.  Looking forward to the improvements.

**Comment by wdrury-uk** [comment]
Thanks @chukka for the update.  We will plan our next internal platform release to coincide few week sprint.  Looking forward to the improvements.

**Comment by chukka** [comment]
@wdrury-uk  
**Update :** We are pushing for a 2.x collection version end of this quarter (delayed)  along with distribution and missionControl roles with postgresql 13.x support

**Comment by chukka** [comment]
Fixed in #106
